apps/financial-aid/api/src/app/modules/auth/auth.service.ts

```diff
@@ -24,6 +24,16 @@ export class AuthService {
     return await res.json()
   }
 
+  async checkUserHistory(nationalId: string): Promise<boolean> {
+    const res = await fetch(
+      `${environment.backend.url}/api/me/?nationalId=${nationalId}`,
+      {
+        headers: { authorization: `Bearer ${environment.auth.secretToken}` },
+      },
+    )
+    return await res.json()
+  }
+
   fakeUser(nationalId: string) {
     const fakeUsers: { [key: string]: User } = {
```

Comment:
Maybe out of scope for this PR, but we should document what fake users we have and how we set which user we want to be. 

We should also probably have a fake user that has applied for the current month.